tests/test_inputs.py

```diff
@@ -199,16 +199,20 @@ def test_positive_negative_input(self):
         assert_raises(ValueError, lambda: inputs.positive(-1))
 
     def test_int_range_good(self):
-        assert_equal(3, inputs.int_range(1, 5, 3, 'my_arg'))
+        int_range = inputs.int_range(1, 5, 'my_arg')
```

Comment:
`int_range` has a default value for the `argument` parameter. Since it's not used in any of these tests, I'd rather remove it to keep things cleaner.